jobserver/models.py

```diff
@@ -241,6 +241,22 @@ def get_repo_url(self):
         f.path.segments += ["tree", self.sha]
         return f.url
 
+    @property
+    def is_invalid(self):
+        """
+        Is this JobRequest invalid?
+
+        JobRequests are a request for a given configuration to be run on a
+        Backend.  That configuration could be unprocessable for a variety of
+        reasons when the Backend looks at it.  We currently surface that to
+        job-server by job-runner creating a Job with the action `__error__`.
+        This property finds Jobs with that action so we can easily see if this
+        particular request was valid or not.
+        """
+        actions = self.jobs.values_list("action", flat=True)
```

Comment:
Minor and pointless optimisation, but would it be better to write this as?
```py
self.jobs.filter(action="__error__").exists()
```

Reply:
Oh, you're using `prefetch_related`, aren't you? So my comment doesn't really apply

Reply:
I am… but who doesn't like a one-liner?!